README.md

```diff
@@ -19,9 +19,9 @@
 	conda install conda-forge::xrootd
 	conda install conda-forge::lightgbm
 	conda install conda-forge::tensorflow
-    conda install pytorch::pytorch
-    conda install conda-forge:alive-progress
-    conda install conda-forge:optuna
+  conda install pytorch::pytorch        # This is not tested to work. If it fails, use `pip install torch`
```

Comment:
I tested this and it worked for me

Reply:
The README file looks like it has missing characters